src/main/scala/io/iohk/ethereum/nodebuilder/StdNode.scala

```diff
@@ -61,7 +64,17 @@ abstract class BaseNode extends Node {
     nodeMetrics.Start.trigger()
   }
 
+  private[this] def logBuildInfo(): Unit = {
+    val map = MantisBuildInfo.toMap
+
+    val json = Serialization.writePretty(map)(DefaultFormats)
```

Comment:
these 2 lines are duplicated above, could they be a function in `io.iohk.ethereum.buildinfo.MantisBuildInfo` instead?

Reply:
I have a better function that uses this `Serialization` and the `DefaultFormats` and turns any (?) object into json, which is generally a needed fucntionality. I will refactor to at least take your comment into consideration.